Content.Client/HUD/GameHud.cs

```diff
@@ -606,18 +609,22 @@ public TopButton(Texture texture, BoundKeyFunction function, IInputManager input
                 );
 
                 ToggleMode = true;
+
+                configManager = IoCManager.Resolve<IConfigurationManager>();
```

Comment:
Pass `IConfigurationManager` in via constructor instead of resolving it like this.

Reply:
also remove the `[Dependency]`-attribute on the field, since its not needed